rts/lra/lra-test/tck/pom.xml

```diff
@@ -168,6 +168,7 @@
                                         <argument>-Djava.net.preferIPv4Stack=true</argument>
                                         <argument>${lra.coordinator.debug.params}</argument>
                                         <argument>${lra.coordinator.trace.params}</argument>
+                                        <argument>-DRecoveryEnvironmentBean.periodicRecoveryPeriod=10</argument>
```

Comment:
should not be considered to change the back off time as well? I mean when periodic recovery is changed from 2min to 10 s then could be the backoff to 3 seconds could be enough? Or do I understand the purpose here wrongly?

Reply:
A backoff of zero is fine so I will update the PR to set that value